core/test/models.cpp

```diff
@@ -6,7 +6,8 @@ using namespace moveit::core;
 
 RobotModelPtr getModel() {
 	// suppress RobotModel errors and warnings
-	ros::console::set_logger_level(ROSCONSOLE_ROOT_LOGGER_NAME ".moveit_core.robot_model", ros::console::levels::Fatal);
+	if (rcutils_logging_set_logger_level("moveit_robot_model.robot_model", RCUTILS_LOG_SEVERITY_ERROR) != RCUTILS_RET_OK)
```

Comment:
hmm this use case seems like we should expose the logger names of MoveIt's core classes. Also, the logger level should be set to fatal, not error.